voter/src/main/java/org/terracotta/voter/options/OptionsParsingImpl.java

```diff
@@ -0,0 +1,62 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Terracotta Core.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.voter.options;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.tc.config.schema.setup.ConfigurationSetupException;
+
+import java.util.List;
+
+@Parameters
+public class OptionsParsingImpl implements OptionsParsing {
+
+  @Parameter(names = {"-help", "-h"}, description = "Help", help = true)
+  private boolean help;
+
+  @Parameter(names = {"-override", "-o"}, description = "Override vote to host:port")
+  private String overrideHostPort;
+
+  @Parameter(names = {"-server", "-s"}, description = "Comma separated host:port to connect to (one per stripe)", splitter = NoCommaSplitter.class)
+  private List<String> serversHostPort;
+
+  @Parameter(names = {"-security-dir", "-srd"}, description = "Security root directory")
```

Comment:
same remarks here as I did before for:
- `-override` => `-override-<something>`
- `-server` => `-connect-to`
- `-security-dir` => `-security-root-directory`